rubocop.yml

```diff
@@ -96,6 +96,10 @@ Style/ClassAndModuleChildren:
     - nested
     - compact
 
+Style/NumericPredicate:
+  # Yes, we think `> 0` is a better choice than `positive?`.
+  EnforcedStyle: comparison
```

Comment:
I completely agree!

Reply:
what about:

```ruby
[1, 2, -3].count(&:positive?)
```

Will this be still accepted by rubocop or not?